pkg/gui/command_status_panel.go

```diff
@@ -0,0 +1,66 @@
+package gui
+
+import (
+	"fmt"
+
+	"github.com/jesseduffield/gocui"
+	"github.com/scbizu/cmd"
+)
+
+// CommandStatus defines the go-cmd's cmd
+type CommandStatus struct {
+	command *cmd.Cmd
+	gui     *gocui.Gui
+}
+
+// NewCommandStatus new command status
+func NewCommandStatus(c *cmd.Cmd, gui *gocui.Gui) *CommandStatus {
+	return &CommandStatus{
+		command: c,
+		gui:     gui,
+	}
+}
+
+// Update updates the command status panel
+func (cs *CommandStatus) Update(ui *Gui) {
```

Comment:
I think this could have a better name, something like `PrintCommandOutput`. The name `Update` suggests to me that it will be an immediate operation